web/src/components/molecules/Asset/Asset/AssetBody/UnzipFileList.tsx

```diff
@@ -65,18 +78,57 @@ const UnzipFileList: React.FC<Props> = ({ file, assetBaseUrl, setAssetUrl, style
   };
 
   return (
-    <Tree
-      switcherIcon={<Icon icon="caretDown" />}
-      expandedKeys={[...expandedKeys]}
-      selectedKeys={[...selectedKeys]}
-      onSelect={onSelect}
-      onExpand={onExpand}
-      treeData={treeData}
-      style={style}
-      multiple={false}
-      showLine
-    />
+    <UnzipFileListWrapper>
+      {archiveExtractionStatus === "in_progress" ? (
+        <Spin tip={t("Decompressing...")} size="large" />
+      ) : archiveExtractionStatus === "failed" ? (
+        <ExtractionFailedWrapper>
+          <ExtractionFailedIcon icon="closeCircle" color="#FF4D4F" size="56px" />
+          <ExtractionFailedText>
+            {t("Failed to decompress, please check the uploaded file.")}
```

Comment:
Failed to decompress. Please check the file and try again.

Reply:
I have already suggested that in the Figma design last Friday, please check the conversation that we had.
![Screenshot 0004-12-06 at 3 36 47 PM](https://user-images.githubusercontent.com/35835415/205839178-b83790e1-89bb-4958-b7bd-73428cd888cd.png)

Reply:
Please let me know what do you think.

Reply:
You're right, I think I will go with `Failed to decompress. Please check the file and try again.`

Reply:
Done!